packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/ActionButton.tsx

```
@@ -1,9 +1,34 @@
+  token?: ITokenInfo;
+  totalValue: number;
+}
+
+export function ActionButton({
+  tradeType,
+  amount,
+  token,
+  totalValue,
+}: IActionButtonProps) {
+  const actionText = tradeType === 'buy' ? 'Buy' : 'Sell';
+  const numericAmount = parseFloat(amount);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)